DN1/test/runtests.jl

```diff
@@ -10,26 +38,19 @@ using Test
   p3 = vrednost(s, 0.0)
   p4 = vrednost(s, 0.5)
 
-  @test p1 == 6.0
+  @test p1 ≈ 6.0
   @test abs(p2 - 5.99) < 0.01
-  @test p3 == 1.0
+  @test p3 ≈ 1.0
   @test p4 >= 1.0 && p4 <= 3.0
 end
 
 @testset "sinus" begin
-  x = range(0, stop=2 * pi, length=5)
+  x = range(0, stop=2 * pi, length=50)
   y = sin.(x)
   s = interpoliraj(x, y)
 
-  p1 = vrednost(s, 0)
-  p2 = vrednost(s, pi / 4)
-  p3 = vrednost(s, pi / 2)
-  p4 = vrednost(s, pi)
-  p5 = vrednost(s, 2 * pi)
-
-  @test p1 == 0.0
-  @test p2 - sqrt(2) / 2 < 1e-15
-  @test p3 ≈ 1.0
-  @test abs(p4) < 1e-15
-  @test abs(p5) < 1e-15
+  diff = [abs(vrednost(s, xi) - sin(xi)) for xi in x]
```

Comment:
Še vedno testirate računanje vrednosti zgolj v interpolacijskih točkah. Naredite nov seznam točk, ki vsebuje tudi notranje točke npr. `x=range(0, stop=2*pi, length=11)` in primerjajte rezultate s funkcijo sin na teh točkah. Število 11 sem izbral, ker je tuje `50` in zato točke ne bodo sovpadale z interpolacijksimi točkami. Napaka mogoče ne bo manjša od `1e-15`, ampak bi morala biti dovolj majhna.

Reply:
Aja seveda, tega pa ne gre ignorirati.